workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/group/CreateGroupExecutor.java

```diff
@@ -0,0 +1,119 @@
+package com.symphony.bdk.workflow.engine.executor.group;
+
+import com.symphony.bdk.ext.group.gen.api.model.BaseProfile;
+import com.symphony.bdk.ext.group.gen.api.model.GroupImplicitConnection;
+import com.symphony.bdk.ext.group.gen.api.model.GroupInteractionTransfer;
+import com.symphony.bdk.ext.group.gen.api.model.GroupVisibilityRestriction;
+import com.symphony.bdk.ext.group.gen.api.model.Member;
+import com.symphony.bdk.ext.group.gen.api.model.Owner;
+import com.symphony.bdk.ext.group.gen.api.model.ReadGroup;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutor;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutorContext;
+import com.symphony.bdk.workflow.swadl.v1.activity.group.CreateGroup;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class CreateGroupExecutor implements ActivityExecutor<CreateGroup> {
+
+  private static final String OUTPUTS_GROUP_KEY = "group";
```

Comment:
It could be suitable to directly have the groupId in outputs (like we do for createRoom and others), in addition to this group object output of course (IMO) WDYT?

Reply:
for the stream/room that's because we can different return types